emulators/mame/Portfile

```diff
@@ -18,7 +18,7 @@ PortGroup           github 1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 github.setup        mamedev mame 0226 mame
-revision            0
+revision            1
```

Comment:
The installed files are not being changed, so don't change the revision.

Reply:
Ah, makes sense Josh. Reverted with the following commit.